snippets/json-schema.code-snippets

```diff
@@ -1,118 +1,118 @@
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"array\",", "\t\"items\": {", "\t\t\"type\": \"number\"", "\t}", "}"],
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"array\",", "\t\"description\": \"${2:Description goes here}\",", "\t\"items\": {", "\t\t\"type\": \"number\"", "\t}", "}${0}"],
         "description": "A numeric array"
     },
-    "String array property": {
+    "Snowboard: String array property": {
         "prefix": ["string array"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"array\",", "\t\"items\": {", "\t\t\"type\": \"string\"", "\t}", "}"],
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"array\",", "\t\"description\": \"${2:Description goes here}\",", "\t\"items\": {", "\t\t\"type\": \"string\"", "\t}", "}${0}"],
         "description": "A string array"
     },
-    "Integer array property": {
+    "Snowboard: Integer array property": {
         "prefix": ["integer array"],
-        "body": ["\"${1:name}\": {", "\t\"type\": \"array\",", "\t\"items\": {", "\t\t\"type\": \"integer\"", "\t}", "}"],
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"array\",", "\t\"description\": \"${2:Description goes here}\",", "\t\"items\": {", "\t\t\"type\": \"integer\"", "\t}", "}${0}"],
         "description": "An array of integers"
     },
-    "Boolean array property": {
+    "Snowboard: Boolean array property": {
         "prefix": ["boolean array"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"array\",", "\t\"items\": {", "\t\t\"type\": \"boolean\"", "\t}", "}"],
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"array\",", "\t\"description\": \"${2:Description goes here}\",", "\t\"items\": {", "\t\t\"type\": \"boolean\"", "\t}", "}${0}"],
         "description": "An array of booleans"
     },
-    "Array of arrays property": {
-        "prefix": ["array"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"array\",", "\t\"items\": {", "\t\t\"type\": \"array\"", "\t}", "}"],
-        "description": "An array of booleans"
+    "Snowboard: Array of arrays property": {
+        "prefix": ["array of arrays"],
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"array\",", "\t\"description\": \"${2:Description goes here}\",", "\t\"items\": {", "\t\t\"type\": \"array\"", "\t}", "}${0}"],
+        "description": "An array of arrays"
     },
-    "Object array property": { // TODO: add the object properties to this
+    "Snowboard: Object array property": {
         "prefix": ["object array"],
-        "body": ["\"${1:name1}\": {", "\t\"type\": \"array\",", "\t\"items\": {", "\t\t\"type\": \"object\"", "\t}", "}"],
-        "description": "An array of booleans"
+        "body": ["\"${1:property_name}\": {", "\t\"type\": \"array\",", "\t\"items\": {", "\t\t\"type\": \"object\",", "\t\t\"description\": \"${2:Description goes here}\",", "\t\t\"properties\": {", "\t\t\t${3}", "\t\t},", "\t\t\"additionalProperties\": ${4:false}", "\t}", "}${0}"],
```

Comment:
Maybe make `${3}` `${0}` instead makes more sense? So you can immediately start defining properties. For additional properties, maybe make that a choice of `true,false`, not sure if that UX is nicer than typing it or not.

Probably the same in the above Object snippets, too.

Reply:
Done.